ros_gym_env_helper:
  ros__parameters:
    env_id: "CartPole-v1"
    env_make_params: {}

agent:
  ros__parameters:
    model_name: "cartpole1"
    replay_memory_size: 100000
    mini_batch_size: 6
    epsilon_init: 1.0
    epsilon_decay: 0.9995
    epsilon_min: 0.05
    network_sync_rate: 10
    learning_rate_a: 0.0001
    discount_factor_g: 0.99
    stop_on_reward: 100000
    fc1_nodes: 10

/**:
  ros__parameters:
    is_training: False
